src/tests/functional/ujs_tests.ts

```diff
@@ -1,29 +1,29 @@
 import { Page, test } from "@playwright/test"
 import { assert } from "chai"
-import { noNextEventOnTarget } from "../helpers/page"
+import { nextEventOnTarget, noNextEventOnTarget } from "../helpers/page"
 
 test.beforeEach(async ({ page }) => {
   await page.goto("/src/tests/fixtures/ujs.html")
 })
 
-test("test clicking a [data-remote=true] anchor within a turbo-frame", async ({ page }) => {
+test("does NOT handle [data-remote=true] anchors within a turbo-frame", async ({ page }) => {
```

Comment:
Maybe this description should be something like "allows UJS to intercept and cancel Turbo requests for anchors inside a turbo-frame"?

Because it's not so much that Turbo doesn't handle those anchors -- normally it would do so. It's that UJS was able to respond first, and cancel the event before Turbo processed it.